.github/workflows/android-fastlane.yml

```diff
@@ -79,9 +79,10 @@ jobs:
           fetch-depth: 0
           ref: ${{ inputs.tag }}
 
-      - uses: maxim-lobanov/setup-xcode@v1
-        with:
-          xcode-version: "14.2"
+      - name: List installations Xcode version
+        run: sudo ls -1 /Applications | grep "Xcode"
+      - name: List used Xcode version
```

Comment:
Note: for Android we do not need a new/different xcode version, we just print it here.